modules/libraries/src/backend/packager.ts

```diff
@@ -0,0 +1,79 @@
+import fse from 'fs-extra'
+import path from 'path'
+import tmp from 'tmp'
+
+import { bpLogger } from '.'
+import { executeNpm } from './utils'
+
+interface Package {
+  name: string
+  scripts: { [scriptName: string]: string }
+  dependencies: { [scriptName: string]: string }
+  bundledDependencies: string[]
+}
+
+const debug = DEBUG('libraries').sub('packager')
+
+const scriptsToDisable = ['publish', 'prepublish', 'postpublish']
+
+const emptyPackage = {
+  name: 'temp',
+  version: '1.0.0',
+  description: '',
+  main: 'index.js',
+  scripts: {},
+  keywords: [],
+  author: '',
+  license: 'ISC'
+}
+
+const disableScripts = (pkg: Package) => {
+  if (!pkg.scripts) {
+    return
+  }
+
+  scriptsToDisable.forEach(script => {
+    if (pkg.scripts[script]) {
+      pkg.scripts[`_${script}`] = pkg.scripts[script]
+      delete pkg.scripts[script]
+    }
+  })
+}
+
+const addBundledDeps = (pkg: Package) => {
+  if (pkg.dependencies) {
+    pkg.bundledDependencies = Object.keys(pkg.dependencies)
+  }
+}
+
+export const packageLibrary = async (name: string, version: string) => {
+  const tmpDir = tmp.dirSync({ unsafeCleanup: true })
+
+  const tempPackageJson = path.join(tmpDir.name, 'package.json')
+  const libFolder = path.join(tmpDir.name, 'node_modules', name)
+  const libPackageJson = path.join(libFolder, 'package.json')
+
+  try {
+    await fse.writeFile(tempPackageJson, JSON.stringify(emptyPackage, undefined, 2))
+
+    // Legacy bundling ensures the library's dependencies are inside the library folder
+    const installResult = await executeNpm(['install', `${name}@${version}`, '--legacy-bundling'], tmpDir.name)
```

Comment:
I have no idea, but since we ship the module with NPM, let's stick to an old version. Otherwise we'd need to re-engineer most of the logic of this module :/